src/nuclio/projects/project/functions/version/version-monitoring/version-monitoring.component.js

```diff
@@ -47,5 +73,20 @@
                 $rootScope.$broadcast('igzWatchWindowResize::resize');
             }, 350);
         }
+
+        //
+        // Private methods
+        //
+
+        /**
+         * Sets actual deploying status in `ctrl.versionStatus`
+         */
+        function setVersionStatus() {
+            if (lodash.isEmpty(lodash.get(ctrl.version, 'ui.deployResult', {}))) {
```

Comment:
No need for the default value `{}` here. If it is not provided then `.get()` will return `undefined` and `.isEmpty(undefined)` is `true`.